src/components/breadcrumbs/crumb/crumb.component.tsx

```diff
@@ -0,0 +1,37 @@
+import { LinkProps } from "components/link/link.component";
+import React from "react";
+import tagComponent, {
+  TagProps,
+} from "../../../__internal__/utils/helpers/tags";
+import { StyledCrumb, Divider } from "./crumb.style";
+
+export interface CrumbProps
+  extends Omit<LinkProps, "toolTipMessage" | "tooltipPosition" | "iconType">,
+    TagProps {
+  /** This sets the Crumb to current, does not render Link */
+  isCurrent?: boolean;
+  /** The href string for Crumb Link */
+  href: string;
+}
+
+const Crumb = React.forwardRef(
+  ({ href, isCurrent, children, ...rest }: CrumbProps) => (
+    <li>
+      <StyledCrumb
+        isCurrent={isCurrent}
```

Comment:
As we have defined `Crumb` as a forward ref, we'll need to specify the `ref` on the root element like so:

```suggestion
const Crumb = React.forwardRef<HTMLLinkElement, CrumbProps>(
  ({ href, isCurrent, children, ...rest }: CrumbProps, ref) => (
    <li>
      <StyledCrumb
       ref={ref}
        isCurrent={isCurrent}
```

So consumers would be able to do:

```tsx
const crumbRef = React.useRef<HTMLLinkElement>(null);
return (
    // ....
    <Crumb ref={crumbRef} href="...">Breadcrumb 1</Crumb>
);
```

Reply:
This is still missing a `ref`